apiserver/facades/client/application/deployrepository.go

```diff
@@ -569,7 +569,7 @@ func (v *deployFromRepositoryValidator) resolveCharm(curl *charm.URL, requestedO
 	}
 
 	// Get the series to use.
-	series, err := selector.CharmSeries()
+	series, _, err := selector.CharmSeries()
```

Comment:
We can't ignore the bool value here. Once this code goes live, the checks in the the juju client won't be made.

Reply:
I changed the implementation a little bit, now it's done in `Validate()`